docs/PC-8800.md

```diff
@@ -44,40 +25,171 @@ At this link you can find information on the BIOS files of the emulator:
 
+This emulator has better joystick support. 
+
+Recommended for :
+
+- Games
+
+
+## BIOS [lr-quasi88]
+
+Place your BIOS files in:
+```
+/home/pi/RetroPie/BIOS/quasi88
+```
+
+## ROMS [lr-quasi88]
+
+Accepted File Extensions: **.d88 (disk) .88d (disk) .cmt (tape) .t88 (tape) .m3u (playlist)** 
+
+Place your PC-88 ROMs in:
+```
+/home/pi/RetroPie/roms/pc88
+```
 
-## Controls
+## Controls [lr-quasi88]
 lr-quasi88 utilises Retroarch configurations.
 
 Add custom retroarch controls to the retroarch.cfg file in
 ```shell
 /opt/retropie/configs/pc88/retroarch.cfg
 ```
-For more information on custom RetroArch controls see: [RetroArch Configuration](RetroArch-Configuration)
\ No newline at end of file
+For more information on custom RetroArch controls see: [RetroArch Configuration](RetroArch-Configuration)
+
+
+## Insert disks in RetroArch menu (2 disk example) [lr-quasi88]:
+(you can also use the joystick in the RetroArch menu, if configured)
+```
+Start the emulator with the first disk
+Press "F1"
+Press "Z-key"
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Load 2-Disk Game"
+Select, with "X-key", the first disk file
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Load 2-Disk Game"
+Select, with "X-key", the second disk file
+Select, with "X-key", "Subsystems"
+Select, with "X-key", "Start 2-Disk Game"
+```
+
+## Multiple disks (M3U playlist) [lr-quasi88]:
+You can create an M3U playlist to easily start the core with multiple disks preloaded instead of using the subsystem interface. Create a text file with the extension ".m3u" and write the filename of each disk on a new line like in the example below.
+```
+# Ys II (Falcom)
+Ys II (Program disk).d88
+Ys II (Disk A).d88
+Ys II (Disk B).d88
+Ys II (User disk).d88
+```
+
+## Cycling between disks [lr-quasi88]:
+If you've loaded multiple disks, you can hold one of the shoulder buttons and use the D-Pad(Arrow-keys) to change the disk that's loaded in each drive. 
+Use L1-shoulder(Q-key) for Drive 1 and R1-shoulder(W-key) for Drive 2. 
+When the shoulder button is released, the chosen disk will be inserted.
+
+## Settings for games [lr-quasi88]:
+[Fire Hawk] and many other games only work if the sub_cpu_mode is always using both CPUs.
+We always kept this setting on, seems that emulation will always benefit.
+Earlier binary emulator versions didn't have this core option.
+So if it doesn't exist in the core options, you have to install the emulator from source.
+You can set this from the RetroArch menu :
+- Press "F1"
+- Goto "Options"
+- Goto "Sub-CPU Mode"
+- Select "Always run both CPUs"
+- Press "F1" 
+- Press "ESC" (your setting are stored)
+
+You can also set the q88_sub_cpu_mode in /opt/retropie/configs/all/retroarch-core-options.cfg :
+```
+q88_sub_cpu_mode = "2"
+```
+
+[Final Crisis] and many other games require a different CPU clock.
+Final Crisis works best on "8 Mhz".
+You can set this from the RetroArch menu :
```

Comment:
_RetroArch menu_ should be renamed to _RetroArch core options_ - add a link to [RetroArch-Core-Options]